client/helpers/map-namespace-description.js

```diff
@@ -21,9 +21,9 @@ export default function(namespace) {
       : 'Unknown',
     'Emit Metrics': emitMetric ? 'Yes' : 'No',
     'History Archival':
-      historyArchivalStatus === 'ENABLED' ? 'Enabled' : 'Disabled',
+      historyArchivalStatus == 'Enabled' ? 'Enabled' : 'Disabled',
```

Comment:
What is the point of this at all? Why not:
```suggestion
      historyArchivalStatus,
```